### ISSUE TITLE
[MiniLLM] About the gradient accumulation in finetune.py

### ISSUE THREAD

**Issue by songmzhang** [microsoft/LMOps#170]
In `finetune.py`, lines283~284, `model.backward(loss)` is directly followed by `model.step()`. 
Does this mean that the model parameters will immediately update once the loss in the current step is back-propogated and the `args.gradient_accumulation_steps` is not working? Is this a bug?

**Comment by t1101675** [comment]
The gradient accumulation is automatically done by [DeepSpeed](https://www.deepspeed.ai/docs/config-json/#batch-size-related-parameters).

**Comment by songmzhang** [comment]
> The gradient accumulation is automatically done by [DeepSpeed](https://www.deepspeed.ai/docs/config-json/#batch-size-related-parameters).

OK, thanks!
